packages/runtime/src/components/builtin/Slot/register.ts

```diff
@@ -0,0 +1,20 @@
+import { lazy } from 'react';
+import { ReactRuntime } from '../../../react'
+import { MakeswiftComponentType } from '../constants'
+import { Checkbox } from '@makeswift/controls'
+import { Slot } from '../../../controls';
+
+export function registerComponent(runtime: ReactRuntime) {
+  return runtime.registerComponent(
+    lazy(() => import('./SlotRoot')),
+      {
+        type: MakeswiftComponentType.Slot, 
+        label: 'Slot',
+        hidden: true,
+        props: {
+          children: Slot(),
+          showFallback: Checkbox({ label: "Use Fallback", defaultValue: true })
```

Comment:
I believe all of our panel labels are in "Sentence case" as opposed to "Title Case"